components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.html

```diff
@@ -14,27 +14,90 @@
-        <div id="left-align">
+        <div id="modal-content-container">
           <form [formGroup]="expressionForm">
+
+              <chef-form-field id="create-type">
+                <label>
+                  <span class="label">Type<span aria-hidden="true">*</span></span>
+                  <select id="expression-type-dropdown"
+                          ngDefaultControl 
+                          (change)="updateFormDisplay('type')"
+                          formControlName="type">
+                    <option value='user'>User</option>
+                    <option value='team'>Team</option>
+                    <option value='token'>Token</option>
+                    <option value='*'>All members (*)</option>
+                  </select>
+                </label>
+                <chef-error *ngIf="expressionForm.get('type').hasError('required') && expressionForm.get('type').touched">
+                  Type is a required field.
+                </chef-error>
+              </chef-form-field>
+
+              <chef-form-field 
+                id="create-identity" 
+                *ngIf="expressionForm.contains('identity')"
+                [@dropInAnimation]="{ value: triggerValue, params: { height: 74.6875 }}" >
+                <label>
+                  <span class="label">Identity Provider<span aria-hidden="true">*</span></span>
+                  <select id="expression-identity-dropdown" 
+                          ngDefaultControl 
+                          (change)="updateFormDisplay('identity')"
+                          formControlName="identity">
+                    <option value="ldap">LDAP</option>
+                    <option value="local">local</option>
+                    <option value="saml">SAML</option>
+                    <option value="*">All {{ allIdentity | titlecase }}s (*)</option>
+                  </select>
+                </label>
+                <chef-error *ngIf="expressionForm.get('identity').hasError('required') && expressionForm.get('identity').touched">
+                  Identity Provider is a required field.
+                </chef-error>
+              </chef-form-field>
+
+              <chef-form-field 
+                id="create-name" 
+                *ngIf="expressionForm.contains('name')"
+                [@dropInAnimation]="{ value: triggerValue, params: { height: 119.427 }}">
+                <label>
+                  <span class="label">{{ nameOrId }}<span aria-hidden="true">*</span></span>
+                  <chef-input id="expression-name-dropdown" 
+                              ngDefaultControl 
+                              (keyup)="updateFormDisplay('name')"
+                              formControlName="name"></chef-input>
+                </label>
+                <p class="note">Wildcards (*) are allowed.</p>
+
+                <chef-error *ngIf="expressionForm.get('name').hasError('required') && expressionForm.get('name').touched">
```

Comment:
Anyone have suggestions for rendering this error?  `chef-input` is not reacting to touched/untouched.  My thought was to instead build one ourselves using an blur event that set a flag in `.component`, but it seems like `chef-input` isn't firing onBlur events either 🤷‍♂

Reply:
As discussed - this will be rolled into https://github.com/chef/automate/issues/1906